EpsilonLights/Core/Src/Lights.c

```diff
@@ -38,19 +38,16 @@ void updateLightsTask(void const* arg)
         /* UPDATE HEADLIGHTS */
         if ((headlightsOff))
         {
-            HAL_GPIO_WritePin(HHIGH_GPIO_Port, HHIGH_Pin, LIGHT_OFF);
-            HAL_GPIO_WritePin(HLOW_GPIO_Port, HLOW_Pin, LIGHT_OFF);
+            HAL_GPIO_WritePin(HEAD_GPIO_Port, HEAD_Pin, LIGHT_OFF);
         }
         else if ((headlightsLow && headlightsHigh))
```

Comment:
Is there still a `headLightsHigh` input? If not I think we should remove all refs to it entirely, rename `headlightsLow`->`headlights` and then make a future task to remove it from the driver controls.